harvester_e2e_tests/scenarios/test_vm_actions.py

```diff
@@ -128,10 +137,17 @@ def _check_vm_instance_migrated():
                                        backuptarget, backup_json)
 
 
+@pytest.mark.backup_and_restore_p2
+@pytest.mark.p2
```

Comment:
Since this is a module which is called from backup restore tests, I don't think we need to mark it